profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyBusinessDisplay.java

```diff
@@ -152,12 +145,10 @@ public void onClick(AjaxRequestTarget target) {
 					// resize iframe
 					target.appendJavaScript("setMainFrameHeight(window.name);");
 				}
-
 			}
-
 		};
-		editButton.add(new Label("editButtonLabel", new ResourceModel(
-				"button.edit")));
+		editButton.add(new Label("editButtonLabel", new ResourceModel("button.edit")));
+		editButton.add(new AttributeModifier("aria-label", new ResourceModel("accessibility.edit.business")));
```

Comment:
This is the way `aria-label` is added to the button. Applied to every Java file, as well as the needed import.